rfc_do_not_merge/RFC.md

```diff
@@ -0,0 +1,268 @@
+# FeatureToggle Component
+
+## Todo
+
+- [ ] What is a feature Flag
```

Comment:
See (Martin Fowler's article)[https://martinfowler.com/articles/feature-toggles.html#De-couplingDecisionPointsFromDecisionLogic].

Feature toggling can be categorized like this:

Experiment: show a beta version of your website to users who subscribed to.

Release: deploy a new version of your code but keep the old one to compare them easily and rollback quickly if needed 

Permission: grant access to a feature for paid accounts using the Security component.

Ops: remove access to a consuming feature if server ressources are low (a k.a. kill switch).